build/loc.proj

```diff
@@ -73,6 +72,16 @@
     <Copy SourceFiles="@(EulaFiles->'%(Identity)')" DestinationFiles="@(EulaFiles->'%(DestinationDir)')"/>
   </Target>
 
+  <Target Name="MoveLcgFilesToArtifacts" AfterTargets="Localize" Condition="'$(BuildRTM)' != 'true'">
+    <ItemGroup>
+      <_MoveLcgFiles Include="@(LocalizedToolFiles)" Condition="'%(LocalizedToolFiles.Extension)'== '.lcg'">
+        <DestinationDir>$(ArtifactsDirectory)localize\PLOC\net\%(LocalizedToolFiles.Filename)%(LocalizedToolFiles.Extension)</DestinationDir>
```

Comment:
don't condition it based on what the target framework contains. instead build the DestinationDir so that lcg files go in a folder structure like:
artifacts\Project-Name\15.0\bin\Debug\TargetFramework\PLOC

When you create a VSTS Build step to copy over *.lcg files using mini-match patterns, it will preserve the relative path and copy to the desired network share location.